### DISCUSSION TITLE
frigate-card bandwidth-based transcoding

### DISCUSSION THREAD

**Discussion by jay-kub** [blakeblackshear/frigate#6267] [Ask A Question]
I use the custom frigate-card in Home Assistant to watch my camera streams remotely, however my upload speed is atrocious and it's practically unusable due to buffering. Is there a way to have the go2rtc stream transcoded based on available bandwidth?

**Comment by NickM-27** [comment]
the frigate card supports substreams so you could select a stream level manually, there is no automatic bandwidith transcoding though

**Reply by jay-kub** [reply]
gotcha, thanks @NickM-27 

I noticed this has been flooding my logs after updating to v.12. Is that normal? Looks like it's just referencing a typical API call but figured I'd ask as I don't recall seeing this previously.

> 2023-04-26 10:41:00.341857306  172.21.0.1 - - [26/Apr/2023:10:41:00 -0600] "GET /api/stats HTTP/1.1" 200 2782 "-" "HomeAssistant/2023.4.6 aiohttp/3.8.4 Python/3.10" "-"

Not sure if it's related to:

> 2023-04-26 10:52:05.281200956  [2023-04-26 10:52:05] frigate.util                   ERROR   : Unable to poll intel GPU stats: ../lib/igt_device_scan.c: 985 scan_drm_devices Failed assertion: !ret

**Reply by NickM-27** [reply]
that's normal from the integration
